packages/components/app/styles/components/table.scss

```diff
@@ -22,6 +22,50 @@ $hds-table-cell-padding-tall: 20px 16px;
   border-spacing: 0;
 }
 
+// TABLE COLGROUP/COL
+.hds-table__col--width-auto {
```

Comment:
@jorytindall do you have design guidance on what column width values we should accept? For now, I went with percentages in multiples of ten (mostly because the mental math is quick to do). 

One thing to note is that since Figma doesn't support relative units, things tend to come out in fixed pixel width instead. We'd expect folks to convert this to relative units in code, though (for a11y reasons). Maybe we could work out some guidance for designers about this, what do you think?

Reply:
If I recall, the approach we had discussed for this was to allow consumers to pass in css of their choosing similar to how we do this for several other components using the `{{style}}` helper behind the scenes. Here's the docs from one of the form components:

<img width="1054" alt="Screen Shot 2022-11-21 at 09 16 06" src="https://user-images.githubusercontent.com/1672302/203090733-007cced2-568c-44db-a625-362fe7640fa3.png">